File: C++/M01_ProgrammingBasics/L04_ForLoop/Lab/Solutions/P10_OddEvenSum.cpp
#include <iostream>
#include <cmath>
#include <algorithm>

using namespace std;

int main() {

	int n;
	cin >> n;

	int even = 0;
	int odd = 0;
	for (int i = 0; i < n; i++) {
		int input;
		cin >> input;
		if (i % 2 == 0) {
			even += input;
		} else {
			odd += input;
		}
	}

	if (even == odd) {
		cout << "Yes" << endl;
		cout << "Sum = " << even << endl;
	} else {
		cout << "No" << endl;
		cout << "Diff = " << abs(even - odd) << endl;
	}

	return 0;
}

File: Python/M01_ProgrammingBasics/L04_ForLoop/Lab/Solutions/P10_OddEvenSum.py
num = int(input())
odd = 0
even = 0

for each in range(num):
    if each % 2 == 1:
        odd += int(input())
    if each % 2 == 0:
        even += int(input())

diff = abs(odd - even)

if odd == even:
    print("Yes")
    print(f"Sum = {odd}")
else:
    print("No")
    print(f"Diff = {diff}")